Signal/src/ViewControllers/ColorPickerViewController.swift

```diff
@@ -46,15 +46,15 @@ class ColorPickerViewController: UIViewController, UIPickerViewDelegate, UIPicke
 
     private let pickerView: UIPickerView
     private let thread: TSThread
-    private let colors: [UIColor]
+    private let colorNames: [String]
```

Comment:
This picker should work in light and dark theme.  Makes more sense for it to be "color name"-based than "color" based.